src/items.zig

```diff
@@ -463,7 +463,7 @@ pub const Tool = struct { // MARK: Tool
 	pub fn deinit(self: *const Tool) void {
 		// TODO: This is leaking textures!
 		//if(self.texture) |texture| {
-			//texture.deinit();
+		//texture.deinit();
```

Comment:
What is this doing here?

Reply:
I have no idea why it still shows in diff, if the master version is identical

Reply:
Alr now its gone!